src/harness.js

```diff
@@ -674,11 +674,12 @@ class Harness {
       resolvedLineage = await this.coreAdapter.buildLineage(resolvedContact._id, stateEnsuringPresenceOfMocks(this.state, user, subject));
     }
 
+    const chtApi = this.coreAdapter.chtScriptApi(this.options.userRoles);
```

Comment:
Can your naming be consistent?  Let's call this thing the same thing everywhere.

```suggestion
    const chtScriptApi = this.coreAdapter.chtScriptApi(this.options.userRoles);
```